docs/conf.py

```diff
@@ -50,9 +51,25 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "sphinx_rtd_theme"
+pycytominer_hex = "#c24dd6"
+html_theme = "furo"
+html_theme_options = {
+    "light_css_variables": {
+        "color-brand-primary": pycytominer_hex,
+        "color-brand-content": pycytominer_hex,
+        "color-api-pre-name": pycytominer_hex,
+        "color-api-name": pycytominer_hex,
+    },
+    "dark_css_variables": {
+        "color-brand-primary": pycytominer_hex,
+        "color-brand-content": pycytominer_hex,
+        "color-api-pre-name": pycytominer_hex,
+        "color-api-name": pycytominer_hex,
+    },
+}
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
```

Comment:
While not a part of the changes, I noticed `sphinx-build` had trouble with this not existing. Consider adding the directory with a `.gitkeep` file or similar or possibly removing this line if it is not needed.

Reply:
I will comment out! It comes standard with sphinx quickstart